src/main/java/net/minecraftforge/fml/common/MinecraftDummyContainer.java

```diff
@@ -33,12 +35,29 @@
     public MinecraftDummyContainer(String actualMCVersion)
     {
         super(new ModMetadata());
-        getMetadata().modId = "Minecraft";
+        getMetadata().modId = "minecraft";
         getMetadata().name = "Minecraft";
         getMetadata().version = actualMCVersion;
         staticRange = VersionParser.parseRange("["+actualMCVersion+"]");
     }
 
+    @Override
+    public boolean isImmutable()
+    {
+        return true;
+    }
+
+    @Override
+    public File getSource()
+    {
+        return new File("minecraft.jar");
```

Comment:
This is technically not really correct, but is the same that is used for the MCP container in the FML loader, so I've just used the same here.